.travis.yml

```diff
@@ -43,20 +43,15 @@ addons:
     - liblapack-dev
     - python-pyexiv2
     - yasm
+before_install:
+  - echo "deb http://archive.ubuntu.com/ubuntu zesty main universe" | sudo tee --append /etc/apt/sources.list
```

Comment:
Zesty going to EOL in January 2018. Soon after that repos will become unavailable and it will be impossible to install ffmpeg, which will break the build.

Reply:
Take a look at the [ubuntu archive](http://archive.ubuntu.com/ubuntu/dists/), [vivid EOL](https://wiki.ubuntu.com/Releases) was in February 4, 2016 and it was not removed.

All ubuntu repositories are available here: https://packages.ubuntu.com
For old or non-supported versions here: http://old-releases.ubuntu.com/ubuntu/dists/

If one day the build breaks, we can fix it changing the `archive.ubuntu.com` to `old-releases.ubuntu.com`

But we only depend on ffmpeg 3.2.4, because [this bug](https://trac.ffmpeg.org/ticket/6294) . As soon as it is fixed, we can move on.